JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R

```diff
@@ -411,7 +424,17 @@ PlotRobustnessSequential <- function(
   return(plot)
 }
 
-
+fixTranslationForExpression <- function(text) {
+  # we should really switch to unicode...
+  text <- gsub("\\s+", "~", trimws(text))
+  idx <- endsWith(text, ":")
+  text[idx] <- paste0("paste(", substring(text[idx], 1, nchar(text[idx]) - 1L), ", ':')")
+  idx <- startsWith(text, ":")
+  text[idx] <- paste0("paste(", "':'", substring(text[idx], 1, nchar(text[idx]) - 1L), ")")
+  text <- gsub("~for", "~'for'", text, fixed = TRUE)
+  text <- gsub("for~", "'for'~", text, fixed = TRUE)
```

Comment:
At this point `for` does not exist any more, right? As it gets translated in `fixTranslationForExpression(gettext("Evidence for H%s"))`